lib/fastly_nsq/listener.rb

```diff
@@ -38,7 +38,7 @@ def initialize(topic:, processor:, preprocessor: FastlyNsq.preprocessor, channel
 
   def call(nsq_message)
     message = FastlyNsq::Message.new(nsq_message)
-    logger.info "[NSQ] Message received on topic [#{topic}]: #{message}"
+    logger.info "topic #{topic}, attempts #{nsq_message.attempts} received: #{message}"
```

Comment:
not that this matter that much but we should use the block format in all these `info`'s as well:
ie: `logger.info { "thing" }`

Reply:
if we are standardizing,  should this not be:
`"topic #{topic}, channel #{channel}, attempts #{nsq_message.attempts}: received #{message}"`